pkg/api/consts.go

```diff
@@ -1,8 +1,9 @@
 package api
 
 const (
-	LatestTag            = "latest"
-	RawTag               = "raw"
-	InternalFlagPrefix   = "internal.operator.dynatrace.com/"
-	AnnotationSecretHash = InternalFlagPrefix + "secret-hash"
+	LatestTag                            = "latest"
+	RawTag                               = "raw"
+	InternalFlagPrefix                   = "internal.operator.dynatrace.com/"
+	AnnotationExtensionsSecretHash       = InternalFlagPrefix + "extensions-secret-hash"
+	AnnotationTelemetryServiceSecretHash = InternalFlagPrefix + "telemetry-service-secret-hash"
```

Comment:
this const you are using only inside pkg/controllers/dynakube/otelc/statefulset/reconciler.go, so IMO better to move it there.

Reply:
[fixed](https://github.com/Dynatrace/dynatrace-operator/pull/4412/commits/36fe9bb91cb57265ab5e0cae9302decef77be599)